tests/client/core/TestInstallAddon.js

```diff
@@ -278,6 +279,17 @@ describe('withInstallHelpers inner functions', () => {
       }));
     });
 
+    it('resets status to uninstalled on onInstallCancelled', () => {
+      const dispatch = sinon.spy();
+      const guid = '{my-addon}';
+      const handler = makeProgressHandler(dispatch, guid);
+      handler({ state: 'STATE_SOMETHING' }, { type: 'onInstallCancelled' });
+      assert(dispatch.calledWith({
```

Comment:
Agreed actually. I copy+pasted this test because it was so similar to another but I too prefer `firstCall`. I'll change it over.